app/routes/application-list.js

```diff
@@ -375,7 +418,8 @@ module.exports = router => {
       applications,
       pagination,
       selectedFilters,
-      hasFilters
+      hasFilters,
+      subjectItems
```

Comment:
Why do we need this?

Reply:
We derive subjects from data and pass them to the view to populate the subject filter.